README.rst

```diff
@@ -34,23 +44,27 @@ Finally, the platform can be run as usual:
 
     tutor local launch
 
+**Please remember:** If the host is rebooted, the AppArmor profile needs to be reloaded.
+
 Configuration
 -------------
 
-For some of these configurations to work correctly, the codejail image must be built again. Command to build codejail: ``tutor images build codejail``.
+To customize the configuration, update the following settings in Tutor:
 
 - ``CODEJAIL_APPARMOR_DOCKER_IMAGE``: (default: ``docker.io/ednxops/codejail_apparmor_loader:latest``)
-- ``CODEJAIL_DOCKER_IMAGE``: (default: ``docker.io/ednxops/codejailservice:14.0.0``)
+- ``CODEJAIL_DOCKER_IMAGE``: (default: ``docker.io/ednxops/codejailservice:{{__version__}}``)
 - ``CODEJAIL_ENFORCE_APPARMOR`` (default: ``True``)
 - ``CODEJAIL_ENABLE_K8S_DAEMONSET`` (default: ``False``)
 - ``CODEJAIL_SKIP_INIT`` (default: ``False``)
 - ``CODEJAIL_SANDBOX_PYTHON_VERSION`` (default: ``3.8.6``)
 - ``CODEJAIL_EXTRA_PIP_REQUIREMENTS`` (optional) A list of pip requirements to add to your sandbox.
     
-    .. code-block:: yaml
+.. code-block:: yaml
 
-        CODEJAIL_EXTRA_PIP_REQUIREMENTS:
-        - pybryt
+    CODEJAIL_EXTRA_PIP_REQUIREMENTS:
+    - pybryt
+        
+**NOTE:** For some configurations to work, the codejail image must be rebuilt. Command to rebuild codejail: ``tutor images build codejail``.
```

Comment:
Do we know which configurations merit a new build? Can we add them here?

Reply:
I think most people don't build this image, so it should rather be framed as what would require a custom build:

1. Additional requirements in the sandbox via `CODEJAIL_EXTRA_PIP_REQUIREMENTS`.
2. A different version of python for the sandbox environment via `CODEJAIL_SANDBOX_PYTHON_VERSION`.
3. Custom version of edx-platform that changes the contents of [`requirements/edx-sandbox`](https://github.com/openedx/edx-platform/tree/open-release/redwood.master/requirements/edx-sandbox).

One plugin that includes a section like this is [`tutor-ecommerce`](https://github.com/overhangio/tutor-ecommerce?tab=readme-ov-file#image-customisation).

Reply:
Implemented :)